lib/db/random.js

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const randomBytes = require('../promise').promisify(require('crypto').randomBytes)
+
+function random(bytes) {
```

Comment:
You won't need the `random(bytes)` or `random.hex()` function anymore if you update the `base32` function to use `randomBytes` directly.